lib/core/src/lib/dialogs/dialog/dialog.component.scss

```diff
@@ -1,3 +1,5 @@
+@import 'styles/mat-selectors';
```

Comment:
this should not be used, we are trying to get rid of this asap

Reply:
used to replace old selectors

Reply:
@DenysVuika so did we already decide what to use instead?

Reply:
@MichalKinas just using proper container styling, in most of the cases there are ways to avoid touching mat components, in the case we definitely need that I suggest creating custom components that wrap that (i.e. we have button, progress already, can have adf-input if needed). But in this PR there's nothing from the workarounds needed

Reply:
removed `mat-selector`. Not sure that now it looks ideal because this overrides now looks more hacking and harder to understand:

- We use `column-gap` but we use `8px` to get `16px` in total with margin from button
- And we use `overflow: hidden` to apply **new** `border-radius` but actually `border-radius` defined in another place: `mdc-dialog__surface`